web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js

```diff
@@ -78,12 +78,15 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
             this._computeColumnAggregates();
             this._computeRowAggregates();
 
+            var defs = [];
+            this.defs = defs;
```

Comment:
`defs` comes from?

Reply:
ping?

Reply:
It is needed, otherwise the following error appears from inheritance:

![image](https://user-images.githubusercontent.com/44768500/75328398-3e25a700-587e-11ea-9846-f36bf073b566.png)

Reply:
@AdriaGForgeFlow can you enable `debug=assets`? Here we don't see where the error happens. I guess it's in `_renderFieldWidget`.
In any case would be nice to mention in a comment that this should be used to wait for all deferred object you want to wait for before rendering but ATM we don't have any.